pkg/virt-controller/watch/vm.go

```diff
@@ -650,6 +653,11 @@ func (c *VMController) startVMI(vm *virtv1.VirtualMachine) error {
 		return nil
 	}
 
+	// Check that PVCs are not used when DataVolmes should
+	if err := handlePVCMisuseInVM(c.pvcInformer, c.recorder, vm); err != nil {
+		return &syncErrorImpl{err, FailedPVCVolumeSourceMisusedReason}
```

Comment:
does using the `syncErrorImpl` make sense here? I think that's only relevant to the vmi controller's sync function. The error reason `FailedPVCVolumeSourceMisusedReason` doesn't mean anything in this context. The VM controller won't use it. 

maybe just `return err` instead?